wizeline/remix-project-lab
# Markdown in project page fixed

# What does this PR do?
In this PR, the WYSIWYG component its rendering correctly the data in the project details page.


# Where should the reviewer start?


`app/routes/projects/$projectId/index.tsx
`
# How should this be manually tested?

1. Fill the WYSIWYG input and submit.
2. See the data in the projects details page.



# Any background context you want to provide?

1. Deleted the "marked-react" library since it was not working.
- https://www.npmjs.com/package/marked-react
2. Added the "html-react-parser" library and used its api to parse the markdown data.
- https://www.npmjs.com/package/html-react-parser


# What are the relevant tickets?



# Screenshots


<img width="854" alt="Screenshot 2022-12-06 at 0 00 10" src="https://user-images.githubusercontent.com/62604183/205762385-104640ac-80df-4a6c-815c-877680040fd5.png">

# Questions



# Checklist
